templates/terraform/objectlib/base.go.erb

```diff
@@ -5,7 +5,7 @@ package google
 <%
     resource_name = product_ns + object.name
     properties = object.all_user_properties
-    api_version = version.base_url.split("/")[-1]
+    api_version = @base_url.split("/")[-1]
```

Comment:
Is this used elsewhere? I feel like base_url should be sprinkled all over the place

Reply:
Yeah, it's a terraform mapper specific thing. It seems like it is used in fetching/parsing discovery docs. This is specifically parsing the base url that exists on the version object to get the version of the api that the resource uses, IE `v1` for most resources, `v1beta4` for sql. 

Saving `base_url` is done only in the terraform mapper to preserve existing functionality that was originally created by passing a version object in place of a string `version_name`